docs/VnVPlan/VnVPlan.tex

```diff
@@ -768,350 +768,386 @@ \subsubsection{Compliance Requirements}
 
 \end{enumerate}
 
-\subsubsection{Operational and Environmental Requirements}
+\subsubsection{Cultural Requirements}
 
-\paragraph{Ability to Connect and Use}
+\paragraph{Cultural}
 
 \begin{enumerate}
-
-  \item{TC-OE-1\\}
+  \item{TC-CR-1\\}
 
     Type: Manual
 
-    Initial State: Device connected to City of Hamilton network
+    Initial State: Github is functional, cloud deployment platform is functional
 
-    Input/Condition: Attempt to connect to the application using the network.
+    Input/Condition: The Github repository code is checked and the code for the
+    display of the user interface is opened.
 
-    Output/Result: The device successfully connects to and is able to
-    use the application.
+    Output/Result: The user interface and user experience is aligned with the 5
+    City of Hamilton cultural pillars. These are Collective Ownership,
+    Steadfast Integrity, Courageous Change, Sensational Service, and
+    Engaged and Empowered Employees.
 
-    How test will be performed: A user on site at the City of
-    Hamilton, Water Division will connect to the network available to
-    employees and contractors and then attempt to connect to and use
-    the application. The test is successful if the application can be
-    connected to using the city's network.
+    How the test will be performed: The capstone team will compare the qualities
+    of the user interface application code with the 5 pillars and assess whether
+    each pillar is met by the application. The test passes if the
+    team determines
+    that the pillars are satisfied.
 
-  \item{TC-OE-2\\}
```

Comment:
an '''\end{enumerate}''' is missing here

Reply:
thanks, fixed